rust/src/nfs/nfs.rs

```diff
@@ -1363,7 +1363,7 @@ impl NFSState {
     }
     fn setfileflags(&mut self, direction: u8, flags: u16) {
         SCLogDebug!("direction: {}, flags: {}", direction, flags);
-        if direction == 1 {
+        if direction == STREAM_TOCLIENT {
```

Comment:
this needs some more to work. Ultimately this is called using
```
rs_nfs_setfileflags(0, state, file_flags);
```
Where direction is passed as 0 or 1.